src/pages/NewChatPage.tsx

```diff
@@ -158,24 +166,32 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, isSearchingF
      * or navigates to the existing chat if one with those participants already exists.
      */
     const createChat = (option: OptionData) => {
-        if (!option.login) {
+        let login = '';
+
+        if (option.login) {
+            login = option.login;
+        } else if (selectedOptions.length === 1) {
+            login = selectedOptions[0].login ?? '';
+        }
+
+        if (!login) {
+            Log.warn('Tried to create chat with empty login');
             return;
         }
-        Report.navigateToAndOpenReport([option.login]);
-    };
 
+        Report.navigateToAndOpenReport([login]);
+    };
     /**
-     * Creates a new group chat with all the selected options and the current user,
-     * or navigates to the existing chat if one with those participants already exists.
+     * Navigates to create group confirm page
      */
-    const createGroup = () => {
-        const logins = selectedOptions.map((option) => option.login).filter((login): login is string => typeof login === 'string');
+    const navigateToConfirmPage = () => {
+        const selectedParticipants = selectedOptions.map((option: OptionData) => ({login: option.login ?? '', accountID: option.accountID ?? -1}));
+        if (personalData && personalData.login) {
```

Comment:
If this is not available we should return early instead of navigating to the confirmation page. It's easier to catch something not working than something barely working